subprojects/language-java/src/main/java/org/gradle/api/tasks/compile/CompileOptions.java

```diff
@@ -60,11 +59,6 @@
 
     private ForkOptions forkOptions = new ForkOptions();
 
-    private boolean useDepend;
-
-    @SuppressWarnings("deprecation")
-    private DependOptions dependOptions = new DependOptions();
-
```

Comment:
I think we can now also remove `isDeprecation()` and `setDeprecation(boolean)` plus the relevant field as are supposedly don't have any more deprecations? I am not sure if these methods were introduced for just that purpose.